Wikipedia/UI-V5/WMFSearchViewController.m

```diff
@@ -410,6 +499,23 @@ - (IBAction)searchForSuggestion:(id)sender {
     [self searchForSearchTerm:self.searchField.text];
 }
 
+- (IBAction)setLanguageOne:(id)sender {
+    [self selectLanguageForButton:sender];
+}
+
+- (IBAction)setLanguageTwo:(id)sender {
+    [self selectLanguageForButton:sender];
+}
+
+- (IBAction)setLanguageThree:(id)sender {
```

Comment:
or do you want them to all call the same action? i can do that… usually don't though…